### DISCUSSION TITLE
[无法更新内容] notion 新写的文章，博客显示不出来

### DISCUSSION THREAD

**Discussion by jaydong2016** [tangly1024/NotionNext#1969] [Q&A]
**描述bug**
从昨天的 408 版本开始，notion 新写的文章，博客显示不出来，读不到，已尝试多次，多次刷新多次写文章，依然发不出文章

更新了 409，情况依旧

**复现步骤**

**Comment by happyzhangyyds** [comment]
我也有此问题
在回退到4.0.5版本中在归档中是可以正常读取到新添加的文章，但是在4.0.8版本中没有正常显示。

**Comment by tangly1024** [comment]
提供一下后台日志看一下

**Comment by jaydong2016** [comment]
> 
我不懂是否是需要这些日志…

![Snipaste_2023-08-02_09-14-18](https://github.com/tangly1024/NotionNext/assets/23270314/2d0c8292-c543-46f9-a3f8-c6e57d6db92c)
![Snipaste_2023-08-02_09-14-59](https://github.com/tangly1024/NotionNext/assets/23270314/dd4273c0-081c-4e64-b1f1-a90c8be019a0)

![Snipaste_2023-08-02_09-16-02](https://github.com/tangly1024/NotionNext/assets/23270314/31f99d68-d8a0-4047-a33a-91f58dd6cffd)

**Comment by priforjm** [comment]
是的，我也遇到相同情况了，v4.06还是正常的

**Comment by Angel0726** [comment]
之前有问题，发现是把博客类型设置成page改回post，问题消失了

**Comment by jimmyhuang22** [comment]
4.0.11 似乎只能加载按照数据库第一个view排序的一部分post页面，无法获取到全部的数据

**Comment by tangly1024** [comment]
> 4.0.11 似乎只能加载按照数据库第一个view排序的一部分post页面，无法获取到全部的数据

是加载所有文章的

**Comment by tangly1024** [comment]
> > 
> 
> 我不懂是否是需要这些日志…
> 
> ![Snipaste_2023-08-02_09-14-18](https://user-images.githubusercontent.com/23270314/257675034-2d0c8292-c543-46f9-a3f8-c6e57d6db92c.png) ![Snipaste_2023-08-02_09-14-59](https://user-images.githubusercontent.com/23270314/257675039-dd4273c0-081c-4e64-b1f1-a90c8be019a0.png)
> 
> ![Snipaste_2023-08-02_09-16-02](https://user-images.githubusercontent.com/23270314/257675076-31f99d68-d8a0-4047-a33a-91f58dd6cffd.png)

后台显示接口全部正常没有报错

**Comment by jimmyhuang22** [comment]
> > 4.0.11 似乎只能加载按照数据库第一个view排序的一部分post页面，无法获取到全部的数据
> 
> 是加载所有文章的

我这边的情况就是只加载前56个文章，后面的不加载，用slug也无法访问
<img width="548" alt="Screenshot 2023-08-03 at 20 13 21" src="https://github.com/tangly1024/NotionNext/assets/52233941/1dcaaa89-f93e-4004-855a-43864106aeb5">

**Comment by jimmyhuang22** [comment]
> > > 4.0.11 似乎只能加载按照数据库第一个view排序的一部分post页面，无法获取到全部的数据
> > 
> > 
> > 是加载所有文章的
> 
> 我这边的情况就是只加载前56个文章，后面的不加载，用slug也无法访问 <img alt="Screenshot 2023-08-03 at 20 13 21" width="548" src="https://user-images.githubusercontent.com/52233941/258108218-1dcaaa89-f93e-4004-855a-43864106aeb5.png">

我新建了一个post页面后，这个问题似乎就修复了，目前加载了全部文章了。
这是个啥情况？

**Comment by jimmyhuang22** [comment]
> > > > 4.0.11 似乎只能加载按照数据库第一个view排序的一部分post页面，无法获取到全部的数据
> > > 
> > > 
> > > 是加载所有文章的
> > 
> > 
> > 我这边的情况就是只加载前56个文章，后面的不加载，用slug也无法访问 <img alt="Screenshot 2023-08-03 at 20 13 21" width="548" src="https://user-images.githubusercontent.com/52233941/258108218-1dcaaa89-f93e-4004-855a-43864106aeb5.png">
> 
> 我新建了一个post页面后，这个问题似乎就修复了，目前加载了全部文章了。 这是个啥情况？

但Menu功能依旧失效，一启用就报错
<img width="941" alt="Screenshot 2023-08-03 at 21 16 32" src="https://github.com/tangly1024/NotionNext/assets/52233941/151bddb6-c706-4a8d-b969-5dbda859a4a1">

heo的更多推荐点开为空
<img width="1197" alt="Screenshot 2023-08-03 at 21 17 52" src="https://github.com/tangly1024/NotionNext/assets/52233941/4f73dad1-e084-4318-8c67-b23b5c34a837">

**Comment by tangly1024** [comment]
感谢反馈关于文章无法显示的问题，提交了相关修复补丁

https://github.com/tangly1024/NotionNext/pull/1416

相关issue https://github.com/tangly1024/NotionNext/issues/1404
